src/Uno.Toolkit.UI/Behaviors/SelectorExtensions.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.System;
+using Windows.UI.ViewManagement;
+using Windows.Foundation.Collections;
+using Uno.Disposables;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using PipsPager = Microsoft.UI.Xaml.Controls.PipsPager;
+#endif
+
+namespace Uno.Toolkit.UI;
+
+/// <summary>
+/// Extensions for <see cref="Selector"/>
+/// </summary>
+[Bindable]
+public static partial class SelectorExtensions
+{
+	readonly static Dictionary<Selector, VectorChangedEventHandler<object>> events = new();
```

Comment:
I'm not sure about this, thinking if this can hold a strong reference of a `Selector Control` and at some point, cause leaks... I'm open to other ideas on how safely remove the event subscription.

Reply:
```suggestion
	readonly static Dictionary<WeakReference<Selector>, VectorChangedEventHandler<object>> events = new();
```